impl/src/test/java/org/ehcache/impl/persistence/DefaultLocalPersistenceServiceTest.java

```diff
@@ -34,25 +35,34 @@
 public class DefaultLocalPersistenceServiceTest {
 
   @Rule
+  public final ExpectedException expectedException = ExpectedException.none();
```

Comment:
I see a duplicate definition of this `expectedException` in the PR ... GitHub rendering issue or real mistake?